test.js

```diff
@@ -0,0 +1,74 @@
+// Iteration 1: Names and Input
+
+let hacker1 = "Elnaz";
+let hacker2 = "Bruno";
+console.log(`The driver's name is ${hacker1}`);
+console.log(`The navigator's name is ${hacker2}`)
+
+// Iteration 2: Conditionals
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+}
+else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+}
+else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters`)
+}
+
+// Iteration 3: Loops
+
+let hacker1Char = "";
+for (let i = 0; i<hacker1.length; i++) {
+    hacker1Char += hacker1[i]+ " ";
+}
+console.log(hacker1Char.toUpperCase())
```

Comment:
You should use the `.toUpperCase()` method inside the loop, otherwise you are not saving the string with the upper cased letters, you are just console logging it:

```
let hacker1Char = "";

for (let i = 0; i<hacker1.length; i++) {
    hacker1Char += hacker1[i].toUpperCase() + " ";
}

console.log(hacker1Char)
```